examples/microprofile/multiport/src/main/resources/application.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+server:
+  port: 8080
+  host: "localhost"
+  sockets:
+    private:
+      port: 8081
+      bind-address: "localhost"
+    admin:
+      port: 8082
+      bind-address: "localhost"
```

Comment:
This syntax  is deprecated. There's a warning in the server start-up output to that effect. The newer syntax is like this, using an array/list with a `name` attribute:
```
server:
  port: 8080
  host: "localhost"
  sockets:
    - name: "private"
      port: 8081
      bind-address: "localhost"
    - name: "admin"
      port: 8082
      bind-address: "localhost"
```

Reply:
Thanks for the catch. Looks like we need to update our docs. I was also going to see if I could do this via microprofile config, but the new syntax does not seem compatible. I'll experiment.

Reply:
Fixed. Used new syntax.